content/doc/tutorials/create-a-pipeline-in-blue-ocean.adoc

```diff
@@ -8,430 +8,294 @@ section: doc
-one of the initial set of tutorials from the link:..[Tutorials overview] page
-first to familiarize yourself with CI/CD concepts (relevant to a technology
-stack you're most familiar with) and how these concepts are implemented in
-Jenkins.
-
-This tutorial uses the same application that the
-link:../build-a-node-js-and-react-app-with-npm[Build a Node.js and React app
-with npm] tutorial is based on. Therefore, you'll be building the same
-application although this time, completely through Blue Ocean. Since Blue Ocean
-provides a simplified Git-handling experience, you'll be interacting directly
-with the repository on GitHub (as opposed to a local clone of this repository).
-
-*Duration:* This tutorial takes 20-40 minutes to complete (assuming you've
-already met the <<prerequisites,prerequisites>> below). The exact duration will
-depend on the speed of your machine and whether or not you've already
-<<run-jenkins-in-docker,run Jenkins in Docker>> from link:..[another tutorial].
+This tutorial shows you how to use the link:/doc/book/blueocean/[Blue Ocean] feature of Jenkins to create a Pipeline that will orchestrate building a simple application.
+
+Before starting this tutorial, be sure to run through at least one of the initial tutorials from the link:..[Tutorials overview] page to familiarize yourself with CI/CD concepts, relevant to the technology stack you're most familiar with, and how these concepts are implemented in Jenkins.
+
+This tutorial uses the same application that the link:../build-a-node-js-and-react-app-with-npm[Build a Node.js and React app with npm] tutorial is based on.
+Therefore, you're building the same application, but this time completely through Blue Ocean.
+Since Blue Ocean provides a simplified Git-handling experience, you'll be interacting directly with the repository on GitHub, as opposed to a local clone.
+
+*Duration:* This tutorial takes 20-40 minutes to complete if you've already met the <<prerequisites,prerequisites>> below. The exact duration will depend on the speed of your machine and whether or not you've already <<run-jenkins-in-docker,run Jenkins in Docker>> from link:..[another tutorial].
 
 You can stop this tutorial at any point in time and continue from where you left
 off.
 
-If you've already run though link:..[another tutorial], you can skip the
-<<prerequisites,Prerequisites>> and <<run-jenkins-in-docker,Run Jenkins in
-Docker>> sections below and proceed on to <<fork-sample-repository,forking the
-sample repository>>. If you need to restart Jenkins, simply follow the restart
-instructions in <<stopping-and-restarting-jenkins,Stopping and restarting
-Jenkins>> and then proceed on.
+If you've already run though link:..[another tutorial], you can skip the <<prerequisites,Prerequisites>> and <<run-jenkins-in-docker,Run Jenkins in Docker>> sections below and proceed on to <<fork-sample-repository,forking the sample repository>>.
```

Comment:
Heads up: if the <<prerequisites, Prerequisites>> formatting is intended to update the display text to lowercase, it is not rendering a such.